extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -32,7 +32,12 @@ export class ProsePreviewPage extends ImportPage {
 	}
 
 	async start(): Promise<boolean> {
-		this.table = this.view.modelBuilder.table().component();
+		this.table = this.view.modelBuilder.table().withProperties<azdata.TableComponentProperties>({
+			data: null,
+			columns: null,
+			fontSize: 32,
```

Comment:
hmm, what unit is this?  `32` seems like a large font.

Reply:
Whoops, this was me testing to see if this is even recognized by slickgrid and it didn't seem to matter.  I'll clean this up.